docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -68,47 +44,83 @@ data:
+You can also modify the default Broker configuration for one or more dedicated namespaces, by defining it in the `namespaceDefaults` section. For example, if you want to use the `config-br-default-channel` ConfigMap for all Brokers by default, but want to use `kafka-channel` ConfigMap for `namespace-1` and `namespace-2`, you would use the following ConfigMap:
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: config-br-defaults
   namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
 data:
-  # Configures the default for any Broker that does not specify a spec.config or Broker class.
   default-br-config: |
     clusterDefault:
       brokerClass: MTChannelBasedBroker
       apiVersion: v1
       kind: ConfigMap
-      name: config-kafka-channel
+      name: config-br-default-channel
       namespace: knative-eventing
     namespaceDefaults:
       namespace-1:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
       namespace-2:
         apiVersion: v1
         kind: ConfigMap
-        name: config-br-default-channel
+        name: kafka-channel
         namespace: knative-eventing
 ```
 
+
+## Configuring the Broker class
+
+Besides configuring the Broker class for each broker [individually](create-broker.md#broker-class-options), it is possible to define the default Broker class cluster wide or on a per namespace basis:
+
+### Configuring the default BrokerClass for the cluster
+
+You can configure the `clusterDefault` Broker class so that any Broker created in the cluster that does not have a `BrokerClass` annotation uses this default class:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+```
+
+### Configuring the default BrokerClass for namespaces
+
+You can modify the default Broker class for one or more namespaces.
+
+For example, if you want to use a `KafkaBroker` class for all other Brokers created on the cluster, but you want to use the `MTChannelBasedBroker` class for Brokers created in `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: KafkaBroker
+    namespaceDefaults:
+      namespace1:
+        brokerClass: MTChannelBasedBroker
+      namespace2:
+        brokerClass: MTChannelBasedBroker
+```
```

Comment:
This approach of specifying only the class while valid usually doesn't work because spec.config is broker specific so users will end up with `class=Kafka` and no spec.config or wrong spec.config.

I know this section was already present but isn't this section reinstating what "Administrator configuration options" is describing?

Reply:
I see your point. I saw the beginning section more as a "short" overview/introduction. This section here should make it more clear for a per-namespace configuration.
Maybe I should add a note, for users specifying only the class without a spec.config...

Reply:
@matzew I know you had some info re broker classes so idk if you have anything to add here.

If this is duplicating content as @pierDipi points out, it might be better to consolidate the content and as I mention above move it to the correct admin docs section